src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1523,6 +1523,15 @@ int xclLogMsg(xclDeviceHandle handle, xrtLogMsgLevel level, const char* tag, con
     return ret;
 }
 
+int xclLog(xrtLogMsgLevel level, const char* tag, const char* format, ...)
+{
```

Comment:
Please compare the log level against the current verbosity level to avoid the va_ stuff.  Ultimately xocl::shim::xclLogMsg does that, but it probably should be done here too.

Reply:
done